packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -891,6 +898,80 @@ class FileCollection extends DocumentCollection {
       data: normalizeFile(resp.data)
     }
   }
+
+  /**
+   * async findNotSynchronizedDirectories - Returns the list of directories not synchronized on the given OAuth client (mainly Cozy Desktop clients) — see https://docs.cozy.io/en/cozy-stack/not-synchronized-vfs/#get-datatypedoc-idrelationshipsnot_synchronizing
+   *
+   * @param  {OAuthClient} oauthClient  A JSON representing an OAuth client, with at least a `_type` and `_id` field.
+   * @param  {object} options           Additional options
+   * @param  {number} options.skip      For skip-based pagination, the number of referenced files to skip.
+   * @param  {number} options.limit     For pagination, the number of results to return.
+   * @param  {object} options.cursor    For cursor-based pagination, the index cursor.
+   * @returns {object}                  The JSON API conformant response.
+   */
+  async findNotSynchronizedDirectories(
+    oauthClient,
+    { skip = 0, limit, cursor } = {}
+  ) {
+    const params = {
+      include: 'files',
+      'page[limit]': limit,
+      'page[cursor]': cursor,
+      sort: 'id'
+    }
+    const url = uri`/data/${oauthClient._type}/${oauthClient._id}/relationships/not_synchronizing`
```

Comment:
In fact, since we use the `/data/io.cozy.oauth.clients` route, shouldn't we create an `OAuthClients` collection?

Reply:
I thought about it but since this is related to files synchronization and I saw the `references` methods which do not pertain only to files either I finally decided to add it to the existing collection.

I'm OK with moving it out as well. I'll listen to your advice on this one as I don't tinker around Cozy Client that much :).

Reply:
> as I don't tinker around Cozy Client that much :)

Shame on you! You should

Reply:
Let's leave the functions here. We need to work on others issues before handling correctly that case!